code/modules/mob/living/carbon/human/say.dm

```diff
@@ -143,4 +235,81 @@
 	return
 
 /mob/living/carbon/human/proc/GetSpecialVoice()
-	return special_voice
\ No newline at end of file
+	return special_voice
+
+
+/*
+   ***Deprecated***
+   let this be handled at the hear_say or hear_radio proc
+   This is left in for robot speaking when humans gain binary channel access until I get around to rewriting
+   robot_talk() proc.
+   There is no language handling build into it however there is at the /mob level so we accept the call
+   for it but just ignore it.
+*/
+
+/mob/living/carbon/human/say_quote(var/message, var/datum/language/speaking = null)
+	var/verb = "says"
+	var/ending = copytext(message, length(message))
+
+	if(speaking)
+		verb = speaking.get_spoken_verb(ending)
+	else
+		if(ending == "!")
+			verb=pick("exclaims","shouts","yells")

```

Comment:
this is a tough one, I'd keep it at exclaims, because often someone who's using "!" isn't necessarily yelling; that said, it would also be nice to have a "yells" flavortext as well. *shrug*